msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -519,7 +519,7 @@ Use the following example to configure your app manifest's `webApplicationInfo`
 
 > [!NOTE]
 >
-> * The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for RSC permission.
+> * If the `ChannelMeeting.ReadBasic.Group` permission is added to the manifest, the bot receives the meeting start or end events automatically from all the channel meetings created where the bot is added.
```

Comment:
For extra clarity, maybe remove the "where" and end the phrase like: "created in all the teams where the bot is added."